dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/SecondaryVoltageControl/Simplified/SecondaryVoltageControl.mo

```diff
@@ -79,6 +83,12 @@ equation
   frozen = FreezingActivated and ((blockedUp and (UpRefPu - UpPu) > 0) or (blockedDown and (UpRefPu - UpPu) < 0));
   switch1.u2 = frozen;
 
+  when (pre(level) <> level) then
```

Comment:
Level is a real so this will basically output at each timestep. Maybe we could use a discrete or a threshold?

Reply:
(Moreover I am not sure the pre of a real is allowed in Modelica)

Reply:
Yes, but I'm not sure I understood what we want to do. The idea is to send a message when the voltage regulation changes ? As it is continuous, I added a message indicating it starts changing and another to indicate it stabilized.

Reply:
Well the model used for DynaWaltz outputs timeline as follows, it would be nice if possible to have something equivalent with this model:
  <event time="421.0000" modelName="RST" message="SVC Area : new level -0.28"/>

Reply:
I changed the timeline condition to match the one present in the SVC of DynawoRTE, however it still uses pre() with a continuous variable (tUpdate), which is not allowed by OpenModelica. Nonetheless, the simulation with Dynawo provides the expected result.